2007-03-13  Mike Frysinger  <vapier@gentoo.org>

	* sysdeps/unix/sysv/linux/arm/ioperm.c: Don't include asm/page.h.

--- ports/sysdeps/unix/sysv/linux/arm/ioperm.c
+++ ports/sysdeps/unix/sysv/linux/arm/ioperm.c
@@ -45,7 +45,6 @@
 #include <sys/mman.h>
 
 #include <linux/version.h>
-#include <asm/page.h>
 #include <sys/sysctl.h>
 
 #define PATH_ARM_SYSTYPE	"/etc/arm_systype"
